modin/data_management/factories/factories.py

```diff
@@ -60,152 +107,147 @@ def get_info(cls) -> FactoryInfo:
-    def _read_clipboard(cls, **kwargs):  # pragma: no cover
+    def _read_clipboard(cls, **kwargs):  # pragma: no cover # noqa
         return cls.io_cls.read_clipboard(**kwargs)
 
     @classmethod
-    def _read_excel(cls, **kwargs):
+    def _read_excel(cls, **kwargs):  # noqa
         return cls.io_cls.read_excel(**kwargs)
 
     @classmethod
-    def _read_hdf(cls, **kwargs):
+    def _read_hdf(cls, **kwargs):  # noqa
         return cls.io_cls.read_hdf(**kwargs)
 
     @classmethod
-    def _read_feather(cls, **kwargs):
+    def _read_feather(cls, **kwargs):  # noqa
         return cls.io_cls.read_feather(**kwargs)
 
     @classmethod
-    def _read_stata(cls, **kwargs):
+    def _read_stata(cls, **kwargs):  # noqa
         return cls.io_cls.read_stata(**kwargs)
 
     @classmethod
-    def _read_sas(cls, **kwargs):  # pragma: no cover
+    def _read_sas(cls, **kwargs):  # pragma: no cover # noqa
         return cls.io_cls.read_sas(**kwargs)
 
     @classmethod
-    def _read_pickle(cls, **kwargs):
+    def _read_pickle(cls, **kwargs):  # noqa
         return cls.io_cls.read_pickle(**kwargs)
 
     @classmethod
-    def _read_sql(cls, **kwargs):
+    def _read_sql(cls, **kwargs):  # noqa
         return cls.io_cls.read_sql(**kwargs)
 
     @classmethod
-    def _read_fwf(cls, **kwargs):
+    def _read_fwf(cls, **kwargs):  # noqa
         return cls.io_cls.read_fwf(**kwargs)
 
     @classmethod
-    def _read_sql_table(cls, **kwargs):
+    def _read_sql_table(cls, **kwargs):  # noqa
         return cls.io_cls.read_sql_table(**kwargs)
 
     @classmethod
-    def _read_sql_query(cls, **kwargs):
+    def _read_sql_query(cls, **kwargs):  # noqa
         return cls.io_cls.read_sql_query(**kwargs)
 
     @classmethod
-    def _read_spss(cls, **kwargs):
+    def _read_spss(cls, **kwargs):  # noqa
         return cls.io_cls.read_spss(**kwargs)
 
     @classmethod
-    def _to_sql(cls, *args, **kwargs):
+    def _to_sql(cls, *args, **kwargs):  # noqa
         return cls.io_cls.to_sql(*args, **kwargs)
 
     @classmethod
-    def _to_pickle(cls, *args, **kwargs):
+    def _to_pickle(cls, *args, **kwargs):  # noqa
         return cls.io_cls.to_pickle(*args, **kwargs)
 
     @classmethod
-    def _to_csv(cls, *args, **kwargs):
+    def _to_csv(cls, *args, **kwargs):  # noqa
         return cls.io_cls.to_csv(*args, **kwargs)
 
 
-class CudfOnRayFactory(BaseFactory):
+@doc(_doc_factory_class, engine_name="`cuDFOnRay")
+class CudfOnRayFactory(BaseFactory):  # noqa: D101
     @classmethod
-    def prepare(cls):
-        """
-        Fills in .io_cls class attribute lazily
-        """
+    @doc(_doc_factory_prepare_method, engine_name="``cuDFOnRayIO``")
+    def prepare(cls):  # noqa: D102
         from modin.engines.ray.cudf_on_ray.io import cuDFOnRayIO
 
         cls.io_cls = cuDFOnRayIO
 
 
-class PandasOnRayFactory(BaseFactory):
+@doc(_doc_factory_class, engine_name="`PandasOnRay")
+class PandasOnRayFactory(BaseFactory):  # noqa: D101
```

Comment:
`D101: missing docstring for public method`, we're inheriting docs here, doc-checker script can't handle that case